salesforce-cdp/1.2/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,71 @@
+= Salesforce Data Cloud Connector 1.2 - Mule 4
+
+
```

Comment:
Extra blank line here

Reply:
Extra lines don't affect the output in HTML.